PowerEditor/src/MISC/PluginsManager/Notepad_plus_msgs.h

```diff
@@ -1025,7 +1025,17 @@ enum Platform { PF_UNKNOWN, PF_X86, PF_X64, PF_IA64, PF_ARM64 };
 	// Return toolbar icon set choice as an integer value. Here are 5 possible values:
 	// 0 (Fluent UI: small), 1 (Fluent UI: large), 2 (Filled Fluent UI: small), 3 (Filled Fluent UI: large) and 4 (Standard icons: small).
 
-	// For RUNCOMMAND_USER
+	#define NPPM_GETNPPSETTINGSDIRPATH (NPPMSG + 119)
+	// int NPPM_GETNPPSETTINGSDIRPATH(size_t strLen, wchar_t *settingsDirPath)

```

Comment:
Please add a note to mention the existence of `NPPM_GETPLUGINSCONFIGDIR` and guide plugin author rather to use it.